.github/workflows/build.yml

```diff
@@ -44,8 +44,8 @@ jobs:
       run: npm run build
 
     - name: Setup
-      run: >
-        echo VERSION=$(node -p "require('./package.json').version") >> $GITHUB_ENV
+      run: |
+        echo "VERSION=$(node -p "require('./package.json').version")" >> $GITHUB_ENV
```

Comment:
Does this still work with the double quotes inside double quotes?

Reply:
In YAML Specs `>` means to fold line ends,
`|` means to keep line ends.

Reply:
> Does this still work with the double quotes inside double quotes?

I see! The inner quotes are in `$(   )` That works in Bash.